Source/WebCore/page/csp/ContentSecurityPolicy.cpp

```diff
@@ -300,6 +302,10 @@ void ContentSecurityPolicy::applyPolicyToScriptExecutionContext()
         }
         if (policy->hasBlockAllMixedContentDirective() && !policy->isReportOnly())
             enableStrictMixedContentMode = true;
+
+        // Intentionally doesn't check for report only, this check is used for performance purposes.
```

Comment:
I don’t think you mean "this check is used for performance purposes". Maybe you mean something more like "this check can be skipped for higher performance"?

Reply:
I'll rewrite that comment to clarify it. I meant this doesn't check for report only because the Boolean this creates is used later for performance purposes not for CSP enforcement.